libs/shared/ui/src/lib/question/question.tsx

```diff
@@ -21,6 +22,23 @@ export function Question(props: QuestionProps) {
     ? `transition-opacity duration-300 delay-700 !opacity-100`
     : `opacity-0 pointer-events-none`
 
+
+  const keyPress = useCallback((e: KeyboardEvent) => {
+    if(e.code === 'KeyL' || e.code === 'KeyT'){
```

Comment:
Why not declaring an array of string including KeyboardEvent code for right and left case, so that it would be easier to add / remove key if needed.

Reply:
Something like that should do the trick 
```suggestion
const keyPress = useCallback(({code}: KeyboardEvent) => {
    const leftKeys = ['KeyL', 'KeyT']
    const rightKeys = ['KeyR', 'KeyB']

    if(leftKeys.includes(code)){
      props.onLeft()
    }else if(rightKeys.includes(code)){
      props.onRight()
    }else if(e.code === 'Space' && props.showResult){
      props.onNext()
    }
  }, [])
```